requirements.txt

```diff
@@ -31,6 +31,7 @@ ipaddress==1.0.22
 iptools==0.6.1
 ipwhois==1.0.0
 itsdangerous==1.1.0
+ivre==0.9.12
```

Comment:
use a local requirements.txt file in your analytics directory inside `contrib` instead of the global one